catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -346,7 +333,16 @@ protected Object executeWithSubject() throws Exception {
   }
 
   private File getInputFile() {
-    final File inputFile = new File(filePath);
+
+    final File inputFile;
+
+    if (StringUtils.isBlank(filePath)) {
+      inputFile = new File(getDefaultIngestLocation());
```

Comment:
❓ What happens if the system property is not set and we get an empty string back?

Reply:
The `inputFile.exists()` check catches it on line 349.

Reply:
It will make for a weird message though ... `File or directory [] must exist`